addons/l10n_bg/data/account_fiscal_position_tax_template.xml

```diff
@@ -0,0 +1,120 @@
+
+            <record id="fp_out_eu_sale_9_personal" model="account.fiscal.position.tax.template">
+                <field name="position_id" ref="fiscal_position_template_out_eu"/>
+                <field name="tax_src_id" ref="l10n_bg_sale_vat_0_exempt"/>
+                <field name="tax_dest_id" ref="l10n_bg_sale_vat_0_export"/>
+            </record>
+            <!-- 0% Exempt [END] -->
+        <!-- Sales [END] -->
+
+        <!-- Purchase -->
+            <!-- 20% Other Tax Credit -->
+            <record id="fp_in_eu_purchase_20_otc" model="account.fiscal.position.tax.template">
+                <field name="position_id" ref="fiscal_position_template_in_eu"/>
+                <field name="tax_src_id" ref="l10n_bg_purchase_vat_20_otc"/>
+                <field name="tax_dest_id" ref="l10n_bg_purchase_vat_0_otc_ica"/>
+            </record>
+
+            <record id="fp_out_eu_purchase_20_otc" model="account.fiscal.position.tax.template">
+                <field name="position_id" ref="fiscal_position_template_out_eu"/>
+                <field name="tax_src_id" ref="l10n_bg_purchase_vat_20_otc"/>
+                <field name="tax_dest_id" ref="l10n_bg_purchase_vat_0_otc_exempt"/>
+            </record>
+            <!-- 20% Other Tax Credit [END] -->
+
+            <!-- 20% Foreign Tax Credit -->
+            <record id="fp_in_eu_purchase_20_ftc" model="account.fiscal.position.tax.template">
+                <field name="position_id" ref="fiscal_position_template_in_eu"/>
+                <field name="tax_src_id" ref="l10n_bg_purchase_vat_20_ftc"/>
+                <field name="tax_dest_id" ref="l10n_bg_purchase_vat_20_ftc_ica"/>
+            </record>
+
+            <record id="fp_out_eu_purchase_20_ftc" model="account.fiscal.position.tax.template">
+                <field name="position_id" ref="fiscal_position_template_out_eu"/>
+                <field name="tax_src_id" ref="l10n_bg_purchase_vat_20_ftc"/>
+                <field name="tax_dest_id" ref="l10n_bg_purchase_vat_20_ftc_exempt"/>
+            </record>
+            <!-- 20% Foreign Tax Credit [END] -->
+
+            <!-- 20% Payable Tax Credit -->
+            <record id="fp_in_eu_purchase_20_ptc" model="account.fiscal.position.tax.template">
+                <field name="position_id" ref="fiscal_position_template_in_eu"/>
+                <field name="tax_src_id" ref="l10n_bg_purchase_vat_20_ptc"/>
+                <field name="tax_dest_id" ref="l10n_bg_purchase_vat_20_ptc_ica"/>
+            </record>
+
+            <record id="fp_out_eu_purchase_20_ptc" model="account.fiscal.position.tax.template">
+                <field name="position_id" ref="fiscal_position_template_out_eu"/>
+                <field name="tax_src_id" ref="l10n_bg_purchase_vat_20_ptc"/>
+                <field name="tax_dest_id" ref="l10n_bg_purchase_vat_20_ptc_exempt"/>
+            </record>
+            <!-- 20% Payable Tax Credit [END] -->
+        <!-- Purchase [END] -->
+    </data>
+</odoo>
```

Comment:
If you declare these lines in xml, why isn't it in all the same file? It looks easier to read to me